tools/export/uvision/__init__.py

```diff
@@ -175,7 +175,7 @@ def format_flags(self):
             '--cpreproc --cpreproc_opts=-D__ASSERT_MSG,' +
             ",".join("-D{}".format(s) for s in
                      self.toolchain.get_symbols(for_asm=True)))
-        flags['asm_flags'] = asm_flag_string
+        flags['asm_flags'] = "{} {}".format(" ".join(self.toolchain.flags['asm']), asm_flag_string)
```

Comment:
@rwalton-arm

Reply:
Given that this change is in the uVision project generator it _shouldn't_ affect the behaviour in the online build service. Meaning we shouldn't have to create another "tools-release" for the online compiler. However, as we've had this PR open since April 2019, let's wait a few more days for @Patater to return before we decide if we're going to accept it. We will need to evaluate if this actually does affect Mbed Studio or the online build service.

Reply:
@Patater can you please make a decision on this?

Reply:
NB: This is mandatory for the on going STM32WL target which is a M4 without FPU.
Without this patch I couldn't compile with Keil IDE. Even if export is no more mandatory for ARM, it still very appreciated from customers.
Thx

Reply:
The Mbed Online IDE actually can export things, so it would affect the Online IDE.

Sorry, we can't risk breaking the online tooling right now. That's why the tools are frozen. Exporter support is important and we should work to support it from the CMake-based build system instead of merging this risky PR.